tests/app/dao/notification_dao/test_notification_dao.py

```diff
@@ -2093,28 +2088,23 @@ def test_dao_get_count_of_letters_to_process_for_today_without_notis_does_not_ra
 
 
 @freeze_time("2017-12-18 17:50")
-def test_dao_get_count_of_letters_to_process_for_date_ignores_service_not_letters_as_pdf(
-        sample_letter_template, sample_template):
-    # not expected
-    create_notification(template=sample_letter_template, created_at='2017-12-18 17:29:00')
+def test_dao_get_count_of_letters_to_process_for_date_ignores_service_not_letters_as_pdf(sample_letter_template):
```

Comment:
Is this test name still relevant or should it renamed for a test for research mode or should there be two different tests?

Reply:
oops i thought i renamed that test 😬yeah it should just be renamed. letters_as_pdf is no longer relevant at all